nautobot/extras/templates/extras/note.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'generic/object_retrieve.html' %}
+{% load buttons %}
+{% load plugins %}
+{% load perms %}
+{% load helpers %}
+
+{% block breadcrumbs %}
+    <li><a href="{% url 'extras:note_list' %}">Notes</a></li>
+    {% if object.assigned_object.get_absolute_url %}
+        <li><a href="{{ object.assigned_object.get_absolute_url }}notes/">{{ object.assigned_object }}</a></li>
+    {% endif %}
+    <li>{{ object }}</li>
+{% endblock breadcrumbs %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-md-5">
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <strong>Note</strong>
+                </div>
+                <table class="table table-hover panel-body attr-table">
+                    <tr>
+                        <td>User</td>
+                        <td>
+                            {{ object.user|default:object.user_name }}
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Object Type</td>
+                        <td>
+                            {{ object.assigned_object_type }}
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Object</td>
+                        <td>
+                            {{ object.assigned_object | hyperlinked_object }}
+                        </td>
+                    </tr>
+                </table>
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <strong>Text</strong>
+                    </div>
+                    <div class="panel-body rendered-markdown">
+                        {{ object.note|render_markdown }}
+                    </div>
```

Comment:
Can you move this panel outside the other panel? This is how it looks now:

![image](https://github.com/nautobot/nautobot/assets/75227981/9abc0ba9-6553-49f7-9a28-2a838e6d2509)

Reply:
Thanks for catching this! Fixed now.